test/e2e-tests.sh

```diff
@@ -72,6 +72,16 @@ publish_test_images || fail_test "one or more test images weren't published"
 # Run the tests
 
 header "Running tests"
-go_test_e2e -timeout=20m ./test/conformance ./test/e2e || fail_test
+
```

Comment:
I suggest the following approach, which is already used in other scripts:

```
failed=0
# Run conformance tests, but don't exit if it fails.
go_test_e2e -timeout=8m ./test/conformance || failed=1

# So that we can also identify failing E2E tests.
go_test_e2e -timeout=12m ./test/e2e || failed=1

(( failed )) && fail_test
```

Reply:
Done